netlogo-gui/src/main/lab/gui/Supervisor.scala

```diff
@@ -74,21 +75,31 @@ class Supervisor(dialog: java.awt.Dialog,
       val fileName = org.nlogo.swing.FileDialog.showFiles(
         workspace.getFrame, "Exporting as spreadsheet", java.awt.FileDialog.SAVE,
         workspace.guessExportName(worker.protocol.name + "-spreadsheet.csv"))
-      addExporter(new SpreadsheetExporter(
+      try{
+	    addExporter(new SpreadsheetExporter(
         workspace.getModelFileName,
         workspace.world.getDimensions,
         worker.protocol,
-        new java.io.PrintWriter(new java.io.FileWriter(fileName))))
+		new java.io.PrintWriter(new java.io.FileWriter(fileName))))
+	  }
+	  catch{
+		case e: IOException => failure(e); return
+      }
```

Comment:
Make sure this closing brace is indented to the same level as the `try/catch` statements.